api/nfd/v1alpha1/types.go

```diff
@@ -139,6 +141,36 @@ type NodeFeatureRuleSpec struct {
 	Rules []Rule `json:"rules"`
 }
 
+// NodeFeatureRuleStatus represents the status of a NodeFeatureRule
+// +kubebuilder:object:root=true
+// +kubebuilder:resource:scope=Namespaced,shortName=nfs
+// +kubebuilder:subresource:status
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +genclient
```

Comment:
These markers were replaced by `kubebuilder:object:root=true`. So, we could drop this off too. Reference: https://book.kubebuilder.io/migration/legacy/migration_guide_v1tov2.html?highlight=genclient#migrate-the-apis